src/uu/wc/src/wc.rs

```diff
@@ -595,20 +647,23 @@ fn wc(inputs: &[Input], settings: &Settings) -> UResult<()> {
         };
         total_word_count += word_count;
         let result = word_count.with_title(input.to_title(&settings.title_quoting_style));
-        if let Err(err) = print_stats(settings, &result, number_width) {
-            show!(USimpleError::new(
-                1,
-                format!(
-                    "failed to print result for {}: {}",
-                    &result.title.unwrap_or_else(|| String::from("<stdin>")),
-                    err,
-                ),
-            ));
+
+        if are_stats_visible {
+            if let Err(err) = print_stats(settings, &result, number_width) {
+                show!(USimpleError::new(
+                    1,
+                    format!(
```

Comment:
is it possible to add a test to cover this error ?

Reply:
Hm, how would you trigger this error in a test? `print_stats` simply writes to `stdout` with

```
writeln!(io::stdout().lock(), "{}", columns.join(" "))  // columns is a Vec of Strings
```